scripts/performance-helper.psm1

```diff
@@ -342,7 +342,7 @@ function Invoke-RemoteExe {
     } -ArgumentList $Exe
 
     if ($Kernel) {
-        $RunArgs = "--kernel $RunArgs"
+        $RunArgs = "--kernel --privateLibrary $RunArgs"
```

Comment:
Another option is have `--kernel` and `--kernelPriv` options.

Reply:
That likely is better. I would likely change the tests at the same time as well so they match. This was pulled straight from the tests.

Reply:
I leave it up to you to do now or later.